src/main/java/org/passay/PasswordGenerator.java

```diff
@@ -124,4 +155,10 @@ protected void randomize(final CharBuffer buffer)
       buffer.put(i, c);
     }
   }
+
+
+  private void validate(final String password, final List<Rule> rules)
```

Comment:
Why is this here?

Reply:
Vestigial code in flight that I forgot to remove. Pushed a fix.